config/initializers/figaro.rb

```diff
@@ -6,7 +6,6 @@
   'domain_name',
   'enable_identity_verification',
   'enable_test_routes',
-  'equifax_ssh_passphrase',
```

Comment:
meaning, if we are reviewing commits to RC that need to be merged back to master, it's clearer than this one should *not* be merged back?